src/LinuxTracing/TracerThread.cpp

```diff
@@ -52,6 +52,7 @@ using orbit_grpc_protos::ThreadNamesSnapshot;
 TracerThread::TracerThread(const CaptureOptions& capture_options)
     : trace_context_switches_{capture_options.trace_context_switches()},
       target_pid_{capture_options.pid()},
+      stack_dump_size_{static_cast<uint16_t>(capture_options.stack_dump_size())},
```

Comment:
Should we `CHECK` that this fits or something like that before casting? Or maybe better, since we don't really want crashes, forcefully bring `capture_options.stack_dump_size()` under a maximum? You could introduce a `kMaxStackSampleUserSize` where `SAMPLE_STACK_USER_SIZE` was and use it for that purpose.

Reply:
Moved the assignment inside the constructor and added the check there. Also created the kMaxStackSampleUserSize. Unfortunately, I can not use it in the CHECKs when reading the flags.